juniper_rocket/src/lib.rs

```diff
@@ -148,7 +164,13 @@ impl FromData for GraphQLRequest {
     type Error = String;
 
     fn from_data(request: &Request, data: Data) -> FromDataOutcome<Self, Self::Error> {
-        if !request.content_type().map_or(false, |ct| ct.is_json()) {
+        // We cannot use `request.content_type()` here.
+        // https://github.com/SergioBenitez/Rocket/issues/505
```

Comment:
I'd like to follow up on https://github.com/SergioBenitez/Rocket/issues/505 to trace the root cause here.

Reply:
Turns out this was a bug in Rocket. The fix is out in rocket v3.6. Let's land this as-is and after https://github.com/graphql-rust/juniper/pull/127 lands I'll post follow-up  to revert to `request.content_type()`

Reply:
https://github.com/graphql-rust/juniper/pull/127 landed so reverting this.